strings/brain_damage_lines.json

```diff
@@ -57,7 +57,8 @@
         "i will snatch erry motherfucker birthday",
         "the ai and borgs are mettacomming I think",
         "u just did the world a little bit more sad place for someone",
-        "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?"
+        "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?",
+        "NYAAAAAA"
```

Comment:
You should have the possibility to have tildes at the end, plus the occasional stuttered N.